components/board/genericlinux/board.go

```diff
@@ -211,8 +226,11 @@ func (sh *spiHandle) Close() error {
 }
 
 func (b *sysfsBoard) I2CByName(name string) (board.I2C, bool) {
-	b.logger.Warn("I2C is not currently supported on sysfs boards.")
-	return nil, false
+	i, ok := b.i2cs[name]
+	if !ok {
+		return nil, false
```

Comment:
Should I log something in this case? I can go either way.

If I shouldn't log, is this function different from just a one-line `return b.i2cs[name]`?